src/utils/locale.ts

```diff
@@ -292,3 +329,98 @@ function getLocale(component: LocalizedComponent): string {
     defaultLocale
   );
 }
+
+interface NumberStringFormatOptions {
+  numberingSystem: NumberingSystem;
+  locale: string;
+  useGrouping: boolean;
+}
+
+/**
+ * This util localizes and de-localizes numbers
```

Comment:
Nitpick: In the future, we should use `formats` and `parses` terminology as these are more common.